superbench/benchmarks/micro_benchmarks/cublas_function/cmd_helper.h

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Utility for storing and parsing command line arguments
+
+#pragma once
+
+#include <fstream>
+#include <iostream>
+#include <limits>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "thirdparty/json.hpp"
+
+using json = nlohmann::json;
+
+// Utility for storing command line arguments
+
+struct Options {
+    int num_test;
+    int warm_up;
+    int num_in_step;
+    std::string para_info_json;
+
+    // Read params of the functions in json from file 'para_file'
+    json read_params() {
+        std::ifstream para_info_fin(para_info_json);
+        json config;
+        if (!para_info_fin) {
+            std::cout << "Error: Open function param file failed." << std::endl;
+            exit(1);
+        } else {
+            para_info_fin >> config;
+            para_info_fin.close();
+            return config;
+        }
+    }
+};
+
+// Utility for parsing command line arguments
+// Referenced by NVIDIA/cutlass/tools/util/include/cutlass/util/command_line.h
+// https://github.com/NVIDIA/cutlass/blob/master/tools/util/include/cutlass/util/command_line.h
+struct CommandLine {
```

Comment:
it's not a good idea copying other project's code directly, maybe you can refer to https://github.com/microsoft/superbenchmark/blob/e977bbc17f1f9da268498408617d8ecf8ad90ead/superbench/benchmarks/micro_benchmarks/kernel_launch_overhead/cuda_kernel_launch.cu#L70